### DISCUSSION TITLE
建议将Remix演示的源代码直接放到每节课程最后

### DISCUSSION THREAD

**Discussion by Frank2333333** [AmazingAng/WTF-Solidity#527] [General]
建议将Remix演示的源代码直接放到每节课程最后，以便于在学习过程中学员能够自己部署体验一下，有利于增强对学习内容的直观理解。现在虽然提供源代码，但需要转到github去找，稍稍有些麻烦，而且新进学员可能不知道。

**Comment by AmazingAng** [comment]
附带个链接可以么？
